packages/mongodb-chatbot-server/src/routes/conversations/conversationsRouter.test.ts

```diff
@@ -200,6 +200,9 @@ describe("Conversations Router", () => {
     await createConversationReq({ app, origin });
     expect(called).toBe(true);
   });
+  it("should create a new conversation with 'null' value for addMessageToConversation if configured", async () => {
+    // TODO: implement
```

Comment:
Do we need to implement this? Looks like it may already be covered in the `addMessageToConversation` test.